src/a11y/index.js

```diff
@@ -1,9 +1,19 @@
-export function getAriaLabel(label) {
-  if (!label) return undefined
+export function getAriaLabel(label, additionalLabelledBy) {
+  const attributes = getAriaAttributeForLabel(label)
 
-  if (label && label[0] === '#') {
-    /* See readme for label. When label starts with # it references ids of dom nodes instead.
-      When used on aria-labelledby, they should be referenced without a starting hash/# */
+  if (additionalLabelledBy) {
+    attributes['aria-labelledby'] = ((attributes['aria-labelledby'] || '') + ' ' + additionalLabelledBy).trim()
+  }
+
+  return attributes
+}
+
+function getAriaAttributeForLabel(label) {
+  if (!label) return {}
+
+  /* See readme for label. When label starts with # it references ids of dom nodes instead.
+    When used on aria-labelledby, they should be referenced without a starting hash/# */
+  if (label[0] === '#') {
     return { 'aria-labelledby': label.replace(/#/g, '') }
   }
   return { 'aria-label': label }
```

Comment:
```suggestion
  return { 'aria-label': label.replace(/^#/g, '') }
```

We can probably forgo the if check too and just do the replace all the time. The comment is descriptive enough.

Reply:
The id check causes different aria-attributed to be outputted. Aria-label is used for text and aria-labelledby is used for references/ids.

Reply:
Ah, sorry, should have looked closely. What about the regex `label.replace(/^#/, '')`? Do you still need to do the global replace?

Reply:
In theory one could have more than one label describing the content, e.g
#refinelabel #categorylabel, so yes it is needed.

Reply:
Not sure I follow. DOM ids start with a hash so you’re trying to replace them: `#someId > someId`. Correct? But the regex you have will replace all hashes like `#some#Id > someId`. Is this intended?

Reply:
You can have multiple label references/ids. If you want to be picky one could replace the regex with label.substring(1).replace(/[ ]#/g but it would still need to be global. If you start of with a hash you probably are not writing a text label though.

Reply:
> But hash is actually allowed in a id in html5 so I guess one should allow it even if highly unlikeley.

I'm not sure how we can support it reliably without introducing more logic to correctly distinguish between id hashes and hashes that are part of the label. 

But when you say 

> You can have multiple label references/ids.

do you mean something like `#myDivOne #myDivTwo`? In that case, `label.substring(1).replace(/ #/g, '')` will be able to handle it without replacing hash in the middle of a label.